tests/system/Storage/RequesterPaysTest.php

```diff
@@ -73,6 +75,48 @@ public function setUp()
         ]);
     }
 
+    public function testBucketSettings()
+    {
+        $bucketName = uniqid(self::TESTING_PREFIX);
+        $objectName = uniqid(self::TESTING_PREFIX);
+        $content = uniqid(self::TESTING_PREFIX);
+
+        // run an http request to call the object's public link and see what we get.
+        $getBody = function($bucket, $object) {
+            $guzzle = new Client;
+
+            // running the same request three times seems to run into some caching mechanism in guzzle.
```

Comment:
I don't think guzzle should be caching anything, I believe it is happening in the API.

https://cloud.google.com/storage/docs/gsutil/addlhelp/WorkingWithObjectMetadata#cache-control

If you set the `Cache-Control` header to private it should disable caching for this object.